WebApplication/ClientApp/src/components/parametersContainer.css

```diff
@@ -108,7 +108,8 @@ div.changedOnUpdate-input-wrapper {
     white-space: pre-wrap;
 }
 
-/* trick to change SVG color. https://stackoverflow.com/a/49627345 */
+/* trick to change SVG color. https://stackoverflow.com/a/49627345 
```

Comment:
Could you please validate that the icon for Update button for cases with adoption warnings is not changed?
This change was done on purpose in #361

Reply:
right, verify it please

Reply:
I have checked it , warning has correct color